cairo/tests/ethereum/cancun/keth/test_body.py

```diff
@@ -0,0 +1,38 @@
+from pathlib import Path
+
+import pytest
+from ethereum_types.numeric import U64
+
+from utils.fixture_loader import load_body_input
+
+pytestmark = pytest.mark.cairo_file(
+    f"{Path().cwd()}/cairo/tests/ethereum/cancun/keth/test_body.cairo"
+)
+
+
+class TestMain:
+    @pytest.mark.parametrize(
+        "zkpi_path",
+        [Path("test_data/22188088.json")],
+    )
+    @pytest.mark.slow
+    def test_body(self, cairo_run, zkpi_path):
+        # Run all transactions of the body
+        body_input = load_body_input(
+            zkpi_path=zkpi_path, start_index=0, chunk_size=int(U64.MAX_VALUE)
```

Comment:
can we add a test where we start at index 6 and chunk size is like 3?

Reply:
I think it's best to do this in the final "end-to-end" test